src/lib/secure_brain.ts

```diff
@@ -0,0 +1,61 @@
+import * as crypto from 'crypto';
+import { Robot } from 'hubot';
+
+class Cipher {
+  private _cipher: crypto.Cipher;
+  private _decipher: crypto.Decipher;
+
+  constructor(private _key: string) { }
+
+  private get cipher(): crypto.Cipher {
+    return this._cipher || (this._cipher = crypto.createCipher('aes256', this._key)); 
```

Comment:
It's for lazy loading the cipher. Now that you point it out though, I realise this will fail in production because `.final()` renders a cipher unusable: 

> Once the cipher.final() method has been called, the Cipher object can no longer be used to encrypt data. Attempts to call cipher.final() more than once will result in an error being thrown.

This should return a new cipher every time.